eya_def_tools/eya_def_tools/data_models/eya_def.py

```diff
@@ -11,30 +11,31 @@
 from eya_def_tools.data_models.base_model import EyaDefBaseModel
 from eya_def_tools.data_models.eya_def_header import (
     Alpha2CountryCode,
+    CommentsField,
```

Comment:
The changes in this file and related changes elsewhere are just renaming to be more consistent with standard conventions.